util/config.go

```diff
@@ -351,21 +366,77 @@ func (config *Config) getAndroidClasspathPath() string {
 	return path.Join(getPlatformSpecificAndroidClasspathPath(config))
 }
 
-func (config *Config) getAndroidNamespacePath() string {
-	gradleFilePath := path.Join(config.AppDir, config.AppTarget, "build.gradle")
-	gradleContent, err := os.ReadFile(gradleFilePath)
+func (config *Config) getAndroidNamespacePath(gradleFileName string) string {
```

Comment:
This might be a bit confusing at the first glance but it's needed to work properly. We used to just cover the case when string was placed directly in the namespace property. In current showcase app however, we use variable with its value declared in the different file instead. Configurator reads the file content but do not read the content of the variables. 

To cover that scenario, we get the name of variable and look for its declaration in the whole project. This may take a while that's why I've added 

```	fmt.Println("Resolving namespace:", identifier, "- scanning project files, this may take a while...")```

once we find it, we get its value and use it as a namespace. 

Sounds a bit crazy but it works properly, I didn't wanted to have this case uncovered.